model/src/lib.rs

```diff
@@ -153,6 +152,7 @@ impl InversePrice {
     }
 }
 
+// TODO: Consider representating leverage as own and counterparty right here
```

Comment:
🙃 A TODO that starts with the word "consider" is a strong candidate for becoming stale imo. I say this because I've done this myself.

Reply:
Uh, that's actually an outdated one that should be removed! If anything, we should pack long/short and not own/counterparty in there :D